plugins.json

```
@@ -596,4 +596,15 @@
     ],
     "dateAdded": "1558342235115"
   }
```

Would you mind adding a comma here?
```suggestion
  },
```